sql/upgrade/upgrade_6.0.0_6.1.0DEV.sql

```diff
@@ -68,6 +68,7 @@ DELETE FROM `role` WHERE `role` IN ('print', 'event_reader', 'full_event_reader'
 
 UPDATE `language` SET `externalid` = 'ada' WHERE langid = 'adb';
 UPDATE `language` SET `externalid` = 'ruby' WHERE langid = 'rb';
+UPDATE `language` SET `externalid` = 'R' WHERE langid = 'r';
```

Comment:
Instead of this, you need an
```
INSERT INTO `language` ...
```

as when the upgrade from 6.0 to 6.1 happens R will not be in the DB yet (because R is introduced in 6.1)

Reply:
I don't fully understand - it is not enough to add r to db in mysql_db_defaultdata.sql script?

Reply:
There are two paths for DOMjudge:

* A fresh installation. This will use `mysql_db_defaultdata.sql`.
* An upgrade from a previous version. This will have used an old version of `mysql_db_defaultdata.sql` which didn't contain R and then apply all `sql/upgrade/upgrade_*` files that are not applied yet in order to get to the latest schema
That is why it is needed in two places.

Reply:
I see, I pushed a fix

Reply:
Could you move it two lines down with an extra newline, so it is its separate block?
Sorry for nitpicking 🙈

Reply:
Sure